src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -111,8 +115,8 @@ const NavHeader = (props) => {
           <View
             style={{
               flexDirection: 'row',
-              alignItems: 'flex-start',
-              paddingTop: hasSearch ? insets.top : insets.top - dimen.normalizeDimen(8),
+              alignItems: 'center',
+              paddingTop: hasSearch ? insets.top : insets.top,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The paddingTop value seems to be the same regardless of the `hasSearch` condition. If this is intentional, you can simplify the code by removing the ternary operator.

```diff
- paddingTop: hasSearch ? insets.top : insets.top,
+ paddingTop: insets.top,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->